### PR TITLE
Add client configs for two parameters

### PR DESCRIPTION
**Pull request by teetak01**

### CODE REVIEW

File: mbed-coap/sn_config.h [hunk 1]
```diff
@@ -98,6 +98,11 @@
  * \brief Sets the CoAP re-send interval in seconds.
  * By default is 10 seconds.
  */
+
+#ifdef MBED_CONF_MBED_CLIENT_DEFAULT_RESPONSE_TIMEOUT
+#define DEFAULT_RESPONSE_TIMEOUT
```
Comment (anttiylitokola) [inline mbed-coap/sn_config.h:103]: Value is missing

Comment (teetak01) [reply mbed-coap/sn_config.h]: doh..fixed :-)

Comment (teetak01) [reply mbed-coap/sn_config.h]: Should we also increase some of the defaults here?

Comment (anttiylitokola) [reply mbed-coap/sn_config.h]: Client is already changing that to 5 seconds at runtime so let's keep it as it is.

**Pull request merged**